docs/quick_start.md

```diff
@@ -26,7 +25,29 @@ cd your-new-task-name
 
 ### 2. Change name and description
 
-Update the `package.json` fields to reflect your app name and description, e.g. `name`, `author`, `repository`
+It's best practice to create a new branch whenever we look to make changes and/or add a new feature. Your IDE may have a way to do this or it can always be done on the command line.
+
+```
+git switch -c your-new-branch-name
```

Comment:
Does this command create the branch if it doesn't exist?

I always do 

`git checkout -b <branch-name>` to Create a new branch and switch to it.

Reply:
I guess @AisOmar could mentioned the different ways to create/switch and touch on the nuances of when the branch starts blank, based on HEAD and so on

Reply:
@mirestrepo Can do!

Reply:
Yeah it's the same basic idea but (from what I've seen), I'll change the language to git checkout